### DISCUSSION TITLE
Migrating S3 destinations

### DISCUSSION THREAD

**Discussion by mockab** [ep1cman/unifi-protect-backup#147] [Q&A] [answered]
First of all, awesome tool and I love it.

I've recently had to switch from one s3 bucket to another.
Got the second connected and uploading net-new clips just fine but it is not uploading the missing clips that were previously uploaded to the old bucket. 
Is there any way to force it to re-upload all clips or check the new destination?

Running via pip & linux service with the following command: `unifi-protect-backup --address <ip> --username <user> --password <password> --no-verify-ssl --rclone-destination s3-destination:/s3-bucket --retention 365d`

**Comment by ep1cman** [comment, marked as answer]
The simplest way to get it to start again is to delete the database file

**Reply by mockab** [reply]
~~Where would the database file be located?~~ 
Edit: asked before I even looked, Thanks!
